.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,45 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+1. - [ ] Update target branch to new version.
+   - For 2.x:
+     - [ ] If releasing 2.x, run auto-update (refer to docs).
+   - For 3.x:
+     - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
+     - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
+       - [ ] Push this local branch upstream and start the release PR for the N version that is to be released.
+1. - [ ] Verify that the `PrivateSourceBuiltArtifactsPackageVersion` in `eng/Versions.props` match N-1 release artifacts
+1. - [ ] Verify that `source-build/ProdConFeed.txt` contains the latest feed required for the release. For info on latest feed (refer OneNote)
+1. - [ ] Verify that we are building with the appropriate stability and release tag (either none for RTM, or "preview1" etc).  This can apply to the SDK itself as well as the core-setup runtime component. Using a local build:
+        
+            dotnet --info
+            dotnet --version
+
+      - [ ] Verify ASP.NET package versions used in templates are correct (dependencies.props may need to be updated)
+      - [ ] Verify MS.NETCore.App packages versions used in downlevel templates are correct (dependencies.props may need to be updated)
+1. - [ ] Complete prebuilt and poison audit. Review the baseline changes to find out if there are any new prebuilts. 
+1. - [ ] Remove new prebuilts, if any. Ideally any new prebuilts that show up in offline builds have to be removed. In some cases, new prebuilts show up in Production builds but gets purged in offline builds automatically as they are not packaged(they may not be needed for the actual build).
+1. - [ ] Stall tagging the release until we have the final validation. Incase of significant delays, notify(PR is ready, waiting on the final validation) the distro maintainer with PR link.
```

Comment:
(Nit) I actually think I'd flip this around to put the action statement (for the checkbox) in the first sentence:

```suggestion
1. - [ ] If significant delays are anticipated, notify the distro maintainer with PR link mentioning that a PR is ready, but we're waiting on the final validation. Do not tag the release until we have the final validation for the Microsoft build.
```